src/MovieCard.js

```diff
@@ -1,15 +1,14 @@
-import React from 'react';
-import './MovieCard.css'
+import React from "react";
+import "./MovieCard.css";
 
-const MovieCard = ({movie}) => {
+const MovieCard = ({ movie, selectMovie }) => {
   return (
-    <article className='movieCard'>
-      <img src={movie.poster_path} className="moviePoster" />
+    <article className="movieCard" onClick={() => selectMovie(movie)}>
+      <img src={movie.poster_path} alt={movie.title} className="moviePoster" />
```

Comment:
Thank you for adding the alt here. I saw react throwing errors about it but didn't get to it before my pr yesterday.